plugins/inputs/mqtt_consumer/sample.conf

```diff
@@ -30,6 +30,13 @@
   ## Connection timeout for initial connection in seconds
   # connection_timeout = "30s"
 
+  ## Interval and ping timeout for keep-alive messages
+  ## The sum of those options defines when a connection loss is detected.
+  ## Note: The keep-alive interval needs to be in second granularity e.g. 1m
+  ##       but not 100ms.
+  # keep_alive = "60s"
+  # ping_timeout = "10s"
+
```

Comment:
I think `startup_error_behavior` should be also mentioned in this example config.